answerking_app/tests/BaseTestClass.py

```diff
@@ -1,6 +1,5 @@
-from abc import abstractmethod
 from datetime import datetime, timedelta
-
+from django.test import TransactionTestCase
```

Comment:
I'm fairly sure that TransactionTestCase should not be used if not dealing with a Database error

Reply:
mh, ok the only difference seems to be the speed

Reply:
I guess you can also remove the tear down and the 2 separate classes in each test file then